articles/active-directory/develop/active-directory-schema-extensions.md

```diff
@@ -59,10 +59,11 @@ For example, here is a claims-mapping policy to emit a single claim from a direc
 
 Where *xxxxxxx* is the appID (or Client ID) of the application that the extension was registered with.
 
+> [!WARNING]
+> When you define a claims mapping policy for a directory extension attribute, use the `ExtensionID` property instead of the `ID` property within the body of the `ClaimsSchema` array, as shown on the example above.
```

Comment:
```suggestion
> When you define a claims mapping policy for a directory extension attribute, use the `ExtensionID` property instead of the `ID` property within the body of the `ClaimsSchema` array, as shown in the example above.
```